.github/workflows/ci.yaml

```diff
@@ -92,6 +92,21 @@ jobs:
         with:
           args: ct lint
 
+      - name: Helm doc generate
+        uses: docker://jnorwood/helm-docs:v1.10.0
+
+      - name: Check if  docs are different
+        run: |
+          dir_dirty=$(git diff HEAD)
+          if [ -z "$dir_dirty" ]; then
+            echo 'Helm-docs generated readmes match chart readmes.'
```

Comment:
Do we need this? I think you can just `if !` and let the step succeed without a message

Also, is there a way to have it part of tasks so we can instruct the user on what command to run to bump?
Also also, I'd change the error message with the command to run to do the bump.

Reply:
Sure, I'll change the if statement.

I can put something similar to it in `tasks.py` if that's desired. The original idea was to just have  it in CI but I can include it in there if desired.

Reply:
@fedepaol if you want me to add it to `tasks.py`, let me know.